modules/migration-DNS-considerations.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+
+[id="migration-DNS-considerations_{context}"]
+= DNS considerations
+
+The DNS domain of the {product-title} 4 target cluster is different from the domain of the {product-title} 3 source cluster. By default, applications get FQDNs of the {product-title} 4 cluster after migration.
+
+If you want to preserve the source DNS domain of migrated applications, consider the following two options.
```

Comment:
"consider" is "fluff". We want the user to do something, not to consider something (= think about it).

To preserve the DNS domain of a migrated application, you can do one of the following options:

* option 1
* option 2